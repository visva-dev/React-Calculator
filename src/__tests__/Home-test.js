import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavBar from '../components/Navbar';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <>
        <BrowserRouter>
          <NavBar />
        </BrowserRouter>
        <h1>Math Magicians</h1>
        <p>
          Lorem ipsum dolor, sit amet consectetur adipisicing elit.
          Consequuntur, quis delectus. Aliquid qui recusandae, maiores tempora
          perspiciatis officiis nobis reprehenderit blanditiis dolor, iure ad
          odit rem iusto, natus necessitatibus perferendis unde veritatis
          consequatur assumenda quo cumque atque ratione alias? Vero molestias
          facere animi dolores pariatur debitis quos at consectetur laudantium,
          similique, itaque, nesciunt veritatis ex?
        </p>
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
