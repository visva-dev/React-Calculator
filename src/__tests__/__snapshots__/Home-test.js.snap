// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="navbar"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      className="nav"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        Home 
      </a>
      <a
        href="/App"
        onClick={[Function]}
      >
        Calculator 
      </a>
      <a
        href="/Quote"
        onClick={[Function]}
      >
        Quote 
      </a>
    </div>
  </div>,
  <h1>
    Math Magicians
  </h1>,
  <p>
    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Consequuntur, quis delectus. Aliquid qui recusandae, maiores tempora perspiciatis officiis nobis reprehenderit blanditiis dolor, iure ad odit rem iusto, natus necessitatibus perferendis unde veritatis consequatur assumenda quo cumque atque ratione alias? Vero molestias facere animi dolores pariatur debitis quos at consectetur laudantium, similique, itaque, nesciunt veritatis ex?
  </p>,
]
`;
